.github/workflows/required.yml

```diff
@@ -0,0 +1,21 @@
+name: Required Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: ['fedora:30', 'fedora:31']
+    steps:
+    - name: Clone the repository
+      uses: actions/checkout@v2
+    - run: docker pull registry.fedoraproject.org/${{ matrix.image }}
+    - run: docker run --name=container --detach -i -v $(pwd):/root/tomcatjss registry.fedoraproject.org/${{ matrix.image }}
+    - run: docker exec container dnf install -y dnf-plugins-core gcc make rpm-build
+    - run: docker exec container dnf copr -y enable ${TOMCATJSS_7_4_REPO:-@pki/master}
+    - run: docker exec container dnf builddep -y --spec /root/tomcatjss/tomcatjss.spec
+    - run: docker exec container dnf remove -y tomcat-native
+    - run: docker exec container /root/tomcatjss/build.sh --with-timestamp --with-commit-id rpm
+    - run: docker exec container rpm -Uvh '/root/build/tomcatjss/RPMS/*'
```

Comment:
Do we want to upload the built rpms as artifacts?

Reply:
Eh, I'm not sure I see the value in this case? It'll just get rebuilt in COPR once the PR merges and TomcatJSS is easy (and fast) to build relative to PKI.

Reply:
ACK